samples/ClassroomLabs/Modules/Library/Tools/LabCreator.ps1

```diff
@@ -267,20 +271,42 @@ $labs = Import-Csv -Path $CsvConfigFile
 Write-Verbose ($labs | Format-Table | Out-String)
 
 $labs | ForEach-Object {
-    if ($_.Emails) {
+
+    # Validate that the name is good, before we start creating labs
+    if (-not ($_.LabName -match "^[a-zA-Z0-9_, '`"!|-]*$")) {
+        Write-Error "Lab Name '$($_.LabName)' can't contain special characters..."
```

Comment:
Adding lab name validation up front to catch any invalid names so we can fail early (rather than fail later)

Reply:
Should this be in the library instead?

Reply:
Thanks!  We agreed offline to keep it here for now - we can always change it later based on feedback...